TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/SourceTargetCaptureTuple.java

```diff
@@ -119,6 +125,26 @@ private JSONObject toJSONObject(SourceTargetCaptureTuple tuple) {
                     .ifPresent(d-> meta.put("targetResponse", jsonFromHttpData(d, tuple.targetResponseDuration)));
             meta.put("connectionId", tuple.uniqueRequestKey);
             Optional.ofNullable(tuple.errorCause).ifPresent(e->meta.put("error", e));
+
+            if (meta.has("sourceResponse") && meta.has("targetResponse")) {
```

Comment:
If there was no response for the target (or even the source), we probably want to still emit the other status code, and especially the part that it doesn't match, right?